SnowSQL/public/Invoke-SnowSql.ps1

```diff
@@ -44,6 +44,12 @@ function Invoke-SnowSql
         [string[]]
         $Query = '!help',
 
+        # Timeout query
```

Comment:
Have this indicate the duration is in seconds.

Reply:
Ok, it's fix